modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -197,6 +203,10 @@ public boolean isUseSetterComments() {
 		return _useSetterComments;
 	}
 
+	public void pomRepositories(String repositoryName, String scope) {
```

Comment:
Technically, we should call this method `pomRepository`, because we're adding a single repository (that is, a single _repository ID - repository URL` pair) to the map. We can have a `pomRepositories` method as well, but I can work on that.